src/components/Prices/index.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { useAuthContext } from '../../util/providers/AuthProvider';
+import "react-responsive-carousel/lib/styles/carousel.min.css";
```

Comment:
Can we keep the quotes consistent here?

Reply:
Oh yea! Try to run `npm run format && npm run lint` before pushing to help automate the cleaning up of your code.

Reply:
Actually, you should run `npm run test:all` to run prettier/typecheck and check out the test coverage too!